tests/gen_expected.sh

```diff
@@ -1,10 +1,12 @@
+#!/usr/bin/env bash
+
 argsFile=$1.args
 if [ -e ${argsFile} ]; then
 	args=$(cat ${argsFile})
 fi
 echo "Args:" ${args}
-dfmt --brace_style=allman ${args} $1.d > allman/$1.d.ref
-dfmt --brace_style=otbs ${args} $1.d > otbs/$1.d.ref
+../bin/dfmt --brace_style=allman ${args} $1.d > allman/$1.d.ref
+../bin/dfmt --brace_style=otbs ${args} $1.d > otbs/$1.d.ref
```

Comment:
Uhm should we put bin onto PATH instead?

Reply:
hm this is just run when the user makes new tests and wants to generate what dfmt currently outputs.

Before it uses the global dfmt which was not necessarily the newest dfmt. I think it's better if it uses the one in this repository path.

Reply:
any other idea? I think it's better than using the dfmt in PATH (because I don't think most users would put their development dfmt on PATH but instead a normal one for IDE usage)

Reply:
This is probably a good change. I think that I originally wrote it this way because had a dfmt symlink in my path that pointed to my checkout of the dfmt code, so on my machines the dfmt on the path was always then one that was just compiled by the testing scripts.